deepchecks/checks/performance/calibration_score.py

```diff
@@ -99,6 +99,7 @@ def _calibration_score(self, dataset: Dataset, model):
                            'probabilistic predictions of a binary classifier are calibrated. It plots the true ' \
                            'frequency of the positive label against its predicted probability, for binned predictions.'
         brier_text = 'The Brier score metric may be used to assess how well a classifier is calibrated. For more ' \
-                     'info, please visit https://en.wikipedia.org/wiki/Brier_score'
+                     'info, please visit <a href="https://en.wikipedia.org/wiki/Brier_score">' \
```

Comment:
target=_blank?

Reply:
it opens in a new tab also without it but i can add it